src/util.ts

```diff
@@ -106,3 +107,9 @@ export class Deferred<T> {
 export function createDeferred<T>(): Deferred<T> {
   return new Deferred()
 }
+
+export function unstrictDecode(value: Buffer) {
+  // rlp library throws on remainder.length !== 0
```

Comment:
Should this an issue of some kind? If so can you open on `rlp`?

Reply:
Ok, maybe we'll find this out along the way when we have more to do with the `devp2p` library and questions like this can benefit from the daily practical experience and some grown intuition. Will leave for now with this comment.